pkgs/by-name/sy/sydbox/package.nix

```
@@ -1,77 +1,89 @@
-    url = "https://git.exherbo.org/${pname}.git/snapshot/${pname}-${version}.tar.xz";
-    sha256 = "0664myrrzbvsw73q5b7cqwgv4hl9a7vkm642s1r96gaxm16jk0z7";
+  src = fetchFromGitLab {
+    domain = "gitlab.exherbo.org";
+    owner = "Sydbox";
+    repo = "sydbox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-9IegNFkOWYt+jdpN0rk4S/qyD/NSPaSqmFnMmCl/3Tk=";
   };
 
+  cargoHash = "sha256-6/D//mkPDRW01SCLmQGWwFCClZ84aJUPhleWGVCJaKM=";
+
   nativeBuildInputs = [
+    mandoc
     pkg-config
-    autoreconfHook
-    python3
-    perl
-    libxslt.bin
-    docbook_xsl
-    docbook_xml_dtd_42
+    scdoc
   ];
 
-  buildInputs = [
-    libseccomp
-  ] ++ lib.optional debugBuild libunwind
-    ++ lib.optionals installTests [
-      gnumake
-      python3
-      perl
-      which
-    ];
+  buildInputs = [ libseccomp ];
 
-  enableParallelBuilding = true;
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
 
-  configureFlags = [ ]
-    ++ lib.optionals installTests [ "--enable-installed-tests"
-      "--libexecdir=${placeholder "installedTests"}/libexec" ]
-    ++ lib.optional debugBuild "--enable-debug";
+  checkFlags = [
+    # rm -rf tmpdir: Os { code: 2, kind: NotFound, message: "No such file or directory" }
+    "--skip=fs::tests::test_relative_symlink_resolution"
+    # Failed to write C source file!: Os { code: 13, kind: PermissionDenied, message: "Permission denied" }
+    "--skip=proc::tests::test_proc_set_at_secure_test_32bit_dynamic"
+    # /bin/false: Os { code: 2, kind: NotFound, message: "No such file or directory" }
+    "--skip=syd_test"
 
-  makeFlags = [ "SYD_INCLUDEDIR=${stdenv.cc.libc.dev}/include" ];
+    # Endlessly stall. Maybe a sandbox issue?
+    "--skip=caps"
+    "--skip=landlock::compat::Compatible::set_compatibility"
+    "--skip=landlock::fs::PathBeneath"
+    "--skip=landlock::fs::PathFd"
+    "--skip=landlock::fs::path_beneath_rules"
+    "--skip=proc::proc_cmdline"
+    "--skip=proc::proc_comm"
+  ];
 
-  doCheck = true;
-  checkPhase = ''
-    # Many of the regular test cases in t/ do not work inside the build sandbox
-    make -C syd check
-  '';
+  # TODO: Have these directories be created upstream similar to the vim files
+  postInstall = ''
+    mkdir -p $out/share/man/man{1,2,5,7}
 
-  postInstall = if installTests then ''
-    moveToOutput bin/syd-test $installedTests
-  '' else ''
-    # Tests are installed despite --disable-installed-tests
-    rm -r $out/bin/syd-test $out/libexec
+    make $makeFlags install-{man,vim}
   '';
 
-  meta = with lib; {
-    homepage = "https://sydbox.exherbo.org/";
+  passthru = {
+    tests.version = testers.testVersion {
+      package = sydbox;
+      command = "syd -V";
+    };
+
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
     description = "seccomp-based application sandbox";
-    license = licenses.gpl2Only;
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ mvs ];
+    homepage = "https://sydbox.exherbo.org/";
```

```suggestion
    homepage = "https://lib.rs/crates/syd";
```

`sydbox.exherbo.org` is dead. <https://sydbox.exherbolinux.org> redirects to <https://lib.rs/crates/syd>.

crates.io refers to <https://man.exherbolinux.org/> as home page, but I feel that the description at lib.rs might be more useful.

---

I think it would be best to just link to the repository if this page and the exherbolinux.org subdomain aren't real homepages, as the manpages and libs.rs are only documentation